server/apple_notification_server.go

```diff
@@ -129,9 +129,20 @@ func (me *AppleNotificationServer) SendNotification(msg *PushNotification) PushR
 		data.Custom("ack_id", msg.AckID)
 	}
 
+	if msg.IsCRTEnabled {
+		data.Custom("is_crt_enabled", "true")
+	} else {
+		data.Custom("is_crt_enabled", "false")
+	}
```

Comment:
why are you adding this as string and not a boolean?

Reply:
https://github.com/mattermost/mattermost-mobile/blob/cc6e68fcce960c0018e2e2140076dd35b11f2915/android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java#L135

https://github.com/mattermost/mattermost-push-proxy/blob/1016c30ba8a5476ab78582f0bb64196bd5da1781/server/push_notification.go#L54

Thought it would be the same for iOS/consistent with iOS.

Reply:
This one should in fact be boolean too